js/content.js

```diff
@@ -148,26 +106,66 @@ function pushComment(text) {
         text = text.substr(0, gMaxTextChars) + gEllipsizedText;
     }
 
-    //console.log(text);
-    myChatsToRender.push( {
-        isNew: true,
-        text: text,
-        time: gTextTime
-    });
+    // Ignore text if TAB is currently not active
+    if (!gTabActive)
```

Comment:
We no longer try to be fancy about tab switching logic. 
When messages arrive while the tab is not focused, we just ignore them completely.